Overrides - show emoji/symbols in override selector and respect override durations as defined in Loop

aug0211 opened the pull request:
Including override symbols (emoji) next to the override name in the custom preset override selector.

aug0211 committed:
Ignore signing files

Added project.pbxproj to .gitignore so that signing is not committed

aug0211 committed:
Revert "Ignore signing files"

This reverts commit facaa2f4051ee9352205f6f8c008a6299f7d4fa4.

aug0211 committed:
Update .gitignore

Ignore signing

aug0211 committed:
Update OverrideView.swift

Added override symbols to OverrideView selector. This allows the user defined override emojis to show in the override selection picker.

aug0211 committed:
Respect preset override durations as defined in Loop

Grab the override duration (sent in seconds), divide by 60 to convert to minutes, and pass in to the call to set an override preset.

Tested for 0 (infinity), 1 hour, 2 hours, and "None" to turn override off.

aug0211 committed:
Update .gitignore

aug0211 committed:
Update TreatmentGraph.swift

Remove rounding for boluses of X.X5 units on Nightscout graph.

gestrich commented on .gitignore:
```diff
@@ -27,3 +27,4 @@ xcuserdata/
 *.playground
 playground.xcworkspace
 timeline.xctimeline
```
I don't think we can add this to the git ignore as this file needs to be tracked as the project changes. I believe you changed this to avoid signing changes. I think the way to resolve that, while tracking other project changes, is to model Loops code signing which uses xcconfig files.

gestrich commented on LoopCaregiver/LoopCaregiver/Views/Actions/OverrideView.swift:
```diff
@@ -29,7 +29,7 @@ struct OverrideView: View {
                         Picker("Overrides", selection: $pickerCurrentlySelectedOverride) {
                             Text("None").tag(nil as NightscoutOverridePreset?)
                             ForEach(overidePresets, id: \.self) { overrideValue in
```
It looks like Loop may use a space between the symbol and name?

gestrich commented on LoopCaregiver/LoopCaregiver/Views/Actions/OverrideView.swift:
```diff
@@ -62,7 +62,7 @@ struct OverrideView: View {
                         
                         do {
                             //TODO: Set appropriate display symbol
```
Great catch here. This is why I like to include the unit in the name -- like durationInSeconds -- although this override preset model comes from Nightscout directly which uses "duration".

aug0211 committed:
Text spacing, bolus & COB decimal precision, Loop icon at top center

• Spacing formatting for home view to give a bit of space at right edge of screen
• Loop icon at top center of HUD, eventually should update this to reflect Loop status (open/closed) and Loop health (red, yellow, green)
• Added assets to support Loop icon
• Placeholder line for active basal rate
• Show more precision in COB and boluses

aug0211 committed:
Merge branch 'aug0211-lcg' of https://github.com/aug0211/LoopWorkspace into aug0211-lcg
